public/javascripts/data/user.js

```diff
@@ -32,15 +33,16 @@ exports.writeUserToDatabase = (username, password) => {
     return;
 };
 
-
+// Function compares user inputted login credentials to database
 exports.tryLogin = (username, password) => {
 
     const db = new Database('public/javascripts/data/stories.db');
+    
     // Check for valid user name
     const userStmt = db.prepare('SELECT user_name FROM users WHERE user_name = (?)');
```

Comment:
You can do your user_name and encrypted_password comparison in 1 SQL statement with the addition of an `AND` condition clause in your `WHERE` clause

Reply:
I was able to make it happen with one clause. I had to do a check before returning back. Details below.

```
let db = new Database('public/javascripts/data/stories.db');
    
    // Get users credentials
    let user = db.prepare('SELECT user_name, encrypted_password FROM users WHERE user_name LIKE 
    (?)').get(username);
    db.close();

    // Must do this check, because express-js will exit with error if trying to access user['user_name'] if user is undefined
    if(user) {
        return (user['user_name'] && 
        hashing.comparePassword(password, user['encrypted_password']));
    };
    
    return false;
```